relayer/src/chain/cosmos.rs

```diff
@@ -336,45 +336,58 @@ impl CosmosSdkChain {
     /// Try to simulate the given tx in order to estimate how much gas will be needed to submit it.
     ///
     /// It is possible that a batch of messages are fragmented by the caller (`send_msgs`) such that
-    /// they do not individually verify. For example for the following batch
+    /// they do not individually verify. For example for the following batch:
     /// [`MsgUpdateClient`, `MsgRecvPacket`, ..., `MsgRecvPacket`]
-    /// if the batch is split in two TX-es, the second one will fail the simulation in `deliverTx` check
+    ///
+    /// If the batch is split in two TX-es, the second one will fail the simulation in `deliverTx` check.
     /// In this case we use the `default_gas` param.
-    fn estimate_gas(&self, tx: Tx) -> Result<u64, Error> {
-        let estimated_gas = self.send_tx_simulate(tx).map(|sr| sr.gas_info);
+    fn simulate_gas(&self, tx: Tx) -> Result<u64, Error> {
```

Comment:
Not sure why the change? The Tx is being "simulated" and it returns the `gas_used`. This is a gas estimation at this point as when the actual Tx is executed the gas might be slightly different.